File: platforms/android/samples/CheckoutKitAndroidDemo/app/src/main/java/com/shopify/checkoutkit/androiddemo/accessibility/AccessibilityIdentifiers.kt
package com.shopify.checkoutkit.androiddemo.accessibility

object AccessibilityIdentifiers {
    const val APP_READY = "checkout-kit-sample-ready"
    const val PRELOAD_STATE_PREFIX = "preload-state-"
    const val PRELOAD_CACHE_HIT_PREFIX = "preload-cache-hit-"

    object Cart {
        const val CHECKOUT_READY = "cart-checkout-ready"
        const val CHECKOUT_BUTTON = "checkout-button"
        const val EMPTY_MESSAGE = "cart-empty-message"
    }

    object Tabs {
        const val CART = "cart-tab"
        const val SETTINGS = "settings-tab"
    }

    object Settings {
        const val CHECKOUT_PRELOADING_TOGGLE = "checkout-preloading-toggle"
    }
}

File: platforms/swift/Samples/CheckoutKitSwiftDemo/CheckoutKitSwiftDemo/Sources/Accessibility/AccessibilityIdentifiers.swift
enum AccessibilityIdentifiers {
    static let appReady = "checkout-kit-sample-ready"
    static let preloadStatePrefix = "preload-state-"
    static let preloadCacheHitPrefix = "preload-cache-hit-"

    enum Cart {
        static let checkoutReady = "cart-checkout-ready"
        static let checkoutButton = "checkout-button"
        static let emptyMessage = "cart-empty-message"
    }

    enum Tabs {
        static let cart = "cart-tab"
        static let settings = "settings-tab"
    }

    enum Settings {
        static let checkoutPreloadingToggle = "checkout-preloading-toggle"
    }
}
